spec/controllers/npq_separation/migration/migrations_controller_spec.rb

```diff
@@ -1,9 +1,56 @@
 require "rails_helper"
 
 RSpec.describe NpqSeparation::Migration::MigrationsController, type: :request do
```

Comment:
I think I caused confusion when adding `request` specs to `spec/controllers`. I've since moved some back to `spec/requests` - should they all be there?

Reply:
For example I created a [sign in as admin](https://github.com/DFE-Digital/npq-registration/blob/main/spec/support/helpers/npq_separation_admin_login.rb#L2) helper which could easily be extended to take a `superadmin` arg (or split into a separate `#sign_in_as_superadmin`)

Reply:
> I think I caused confusion when adding `request` specs to `spec/controllers`. I've since moved some back to `spec/requests` - should they all be there?

Yeah I think that would make more sense. Probably worth doing all of them in a separate PR though. I'd also like to move `spec/lib/services` to `spec/services` to reflect the app structure as well.

Reply:
> For example I created a [sign in as admin](https://github.com/DFE-Digital/npq-registration/blob/main/spec/support/helpers/npq_separation_admin_login.rb#L2) helper which could easily be extended to take a `superadmin` arg (or split into a separate `#sign_in_as_superadmin`)

That's nicer than mocking `current_admin`, I'll update 👍